### ISSUE TITLE
Upsampling for modified FCN

### ISSUE THREAD

**Issue by pranavgundewar** [DeepSegment/ConvNet-Segment#15]
Hi!

Thanks for sharing your work. I have modified given Inception v3 FCN architecture and now I have to upsample the logits to the original dimension. I did not understand upsampling part of your code specifically why did you use [31, 31] and stride 8 for logits.

Please could you explain the logic?
